dm/dm/master/scheduler/scheduler.go

```diff
@@ -2193,6 +2225,25 @@ func (s *Scheduler) tryBoundForSource(source string) (bool, error) {
 	return true, nil
 }
 
+// tryBoundForSourceWorker tries to bound a source to the specified worker.
+// returns (true, nil) after bounded.
+func (s *Scheduler) tryBoundForSourceWorker(source string, w *Worker) (bool, error) {
```

Comment:
"SourceWorker" is a terminology used elsewhere, so I prefer we rename it. My suggestion is `tryBoundSourceToWorker`